src/main/java/com/powsybl/networkconverterserver/NetworkConverterController.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.networkconverterserver;
+
+import com.powsybl.networkconverterserver.dto.NetworkIds;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+/**
+ * @author Abdelsalem Hedhili <abdelsalem.hedhili at rte-france.com>
+ */
+
+@RestController
+@RequestMapping(value = "/" + NetworkConverterConstants.IIDM_CONVERTER_API_VERSION + "/")
+@Api(tags = "iidm-converter-server")
+public class NetworkConverterController {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(NetworkConverterController.class);
+
+    @Autowired
+    private NetworkConverterService networkConverterService;
+
+    @PostMapping(value = "/persistent-store/{caseName}")
```

Comment:
I am not sure what should be a good REST API for that.
/cases/{caseName}/to-network ???